harvard-lts/CURIOSity
# Adding sequence variable to embed.

**Adding sequence variable to embed.**
* * *

# What does this Pull Request do?
This checks the manifest URL of an item to see if it has the `n` request variable. If it does it adds that variable as a parameter to the embed call.

# How should this be tested?

A description of what steps someone could take to:
* This cannot be tested until mps-embed has been deployed to Dev/QA.
* Rebuild CURIOSity off of the `fix-sequence` branch
* Create and Harvest the Botanical Illustrations exhibit if you do not already have it locally.
* Search for this item: "Plate 3. Epimedium grandiflorum ; Alfred Riocreux botanical watercolors"
* Click on the item and confirm that plate 3 displays in the legacy viewer.
* You can do something similar for any other objects that should start on a different plate than the first one.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @f8f8ff
